src/test/java/test/listeners/github2385/IssueTest.java

```diff
@@ -0,0 +1,113 @@
+package test.listeners.github2385;
+
+import org.testng.TestNG;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.xml.XmlPackage;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+import test.SimpleBaseTest;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.AssertJUnit.assertTrue;
```

Comment:
Wrong assertTrue ;)

Reply:
OK, I have fixed this. Thank you very much for your review and I'm glad to contribute to the project.